src/sentry/static/sentry/app/views/admin/adminProjects.tsx

```diff
@@ -1,32 +1,52 @@
-/* eslint-disable react/jsx-key */
 import React from 'react';
+import {RouteComponentProps} from 'react-router';
 import moment from 'moment';
 
 import ResultGrid from 'app/components/resultGrid';
 import {t} from 'app/locale';
+import {Organization} from 'app/types';
+import AsyncView from 'app/views/asyncView';
 
-export const prettyDate = function (x) {
-  return moment(x).format('ll');
+const prettyDate = (x: string) => moment(x).format('ll');
```

Comment:
maybe we should rename x to date?

Reply:
It's used only once I'll move it inline, it'll be simpler